src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/PasswordPolicy.cs

```diff
@@ -0,0 +1,202 @@
+        /// <exception cref="TimeoutException">Thrown when failed because of timeout.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when application does not have privilege to access this method.</exception>
+        public int GetMaxInactivityTimeDeviceLock()
+        {
+            int value;
+            int ret = Interop.DevicePolicyManager.PasswordGetMaxInactivityTimeDeviceLock(_dpm.GetHandle(), out value);
+            if (ret != (int)Interop.DevicePolicyManager.DpmError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get password maximum inactivity time policy " + ret);
+                throw DevicePolicyManagerErrorFactory.GetException(ret);
+            }
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets maximum number of failed attempts before device is wiped.
+        /// If user fails the last attempt, device will be wiped.
+        /// </summary>
+        /// <returns>Maximum count for failed passwords.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/dpm.password</privilege>
+        /// <privlevel>partner</privlevel>
+        /// <exception cref="ArgumentException">Thrown when failed because of invalid handle of DevicePolicyManager.</exception>
+        /// <exception cref="TimeoutException">Thrown when failed because of timeout.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when application does not have privilege to access this method.</exception>
+        public int GetMaximumFailedAttemptsForWipe()
+        {
+            int value;
+            int ret = Interop.DevicePolicyManager.PasswordGetMaximumFailedAttemptsForWipe(_dpm.GetHandle(), out value);
+            if (ret != (int)Interop.DevicePolicyManager.DpmError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get password maximum failed attempts policy " + ret);
+                throw DevicePolicyManagerErrorFactory.GetException(ret);
+            }
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets minimum complex char in password.
+        /// Complex characters are all non-alphabetic characters; that is, numbers and symbols.
+        /// </summary>
+        /// <returns>Number of minimum complex char in password.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/dpm.password</privilege>
+        /// <privlevel>partner</privlevel>
+        /// <exception cref="ArgumentException">Thrown when failed because of invalid handle of DevicePolicyManager.</exception>
+        /// <exception cref="TimeoutException">Thrown when failed because of timeout.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when application does not have privilege to access this method.</exception>
+        public int GetMinComplexChars()
```

Comment:
How about rename to MinimumRequiredComplexChars
```suggestion
        public int MinimumRequiredComplexChars { get { ....  }; }
```